src/Symfony/Component/Form/ChoiceList/Factory/CachingFactoryDecorator.php

```diff
@@ -116,8 +116,16 @@ public function createListFromLoader(ChoiceLoaderInterface $loader, $value = nul
     /**
      * {@inheritdoc}
      */
-    public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, $index = null, $groupBy = null, $attr = null)
+    public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, $index = null, $groupBy = null, $attr = null/* , $skipLabelDeprecation = false */)
     {
+        if (false === $label) {
+            if (@func_get_arg(6)) {
```

Comment:
We should show the warning by default, unless you pass `false` explicitly. See an example in OptionsResolver component.